runtime-sdk/src/modules/consensus_accounts/types.rs

```diff
@@ -2,14 +2,21 @@
 use crate::types::{address::Address, token};
 
 /// Deposit into runtime call.
+/// Transfer from consensus staking to an account in this runtime.
+/// The transaction signer has a consensus layer allowance benefiting this runtime's staking
+/// address. The `to` address runtime account gets the tokens.
 #[derive(Clone, Debug, cbor::Encode, cbor::Decode)]
 pub struct Deposit {
+    pub to: Option<Address>,
```

Comment:
Add `#[cbor(optional)]` here to allow the field to be omitted.

Reply:
ok